.envrc

```diff
@@ -2,11 +2,12 @@ export DOCKER_HOST_IP=${DOCKER_HOST_IP:-127.0.0.1}
 export NETWORK=regtest
 export NODE_ENV=development
 
-export LND1_TLS=$(base64 dev/lnd/tls.cert | tr -d '\n\r')
-export LND2_TLS=$(base64 dev/lnd/tls.cert | tr -d '\n\r')
-export LNDONCHAIN_TLS=$(base64 dev/lnd/tls.cert | tr -d '\n\r')
-export TLSOUTSIDE1=$(base64 dev/lnd/tls.cert | tr -d '\n\r')
-export TLSOUTSIDE2=$(base64 dev/lnd/tls.cert | tr -d '\n\r')
+LND_TLS=$(base64 ~/.lnd/tls.cert | tr -d '\n\r')
+export LND1_TLS=${LND_TLS:-$(base64 dev/lnd/tls.cert | tr -d '\n\r')}
+export LND2_TLS=${LND_TLS:-$(base64 dev/lnd/tls.cert | tr -d '\n\r')}
```

Comment:
you may need only one TLS here (ie: not needed to set LND2_TLS), although I'm not sure how the backend would react. this may need some changes in the yaml also

the general idea here is that our backend can work with multiple lnd, and we run 2 by default, but on a raspiblitz we can just depend on one

I'm not recommending looking at it now, but after there is a first end-to-end integration working

Reply:
yes, currently just using the tls.cert and the macaroon of the one LND instance if it exists. Alternatives could be added for other bases as well in the longer term.
I there is no local file it should fall back to the docker container (in the case of the macaroons) or the predefined value in case of the TLS cert here.